parse_test.go

```diff
@@ -0,0 +1,48 @@
+package gobl_test
+
+import (
+	"testing"
+
+	"github.com/invopop/gobl"
+	"github.com/invopop/gobl/note"
+	"github.com/stretchr/testify/assert"
+)
+
+var parseExampleDoc = `{
+	"$schema": "https://gobl.org/draft-0/note/message",
```

Comment:
Can we upload this schemas to the _gobl.org_ website? It will help code editors when opening a GOBL file.

Reply:
I haven't been able to figure out how to do it yet! The gobl.org site is auto-generated, but ideally we want HTML or JSON responses depending on the `Accept` header in the request... I'll see if we can at least provide JSON responses for the time being, ignoring the file extension of the source.

Reply:
Bit of a hack, but this is now live!